src/api/storage-pools.tsx

```diff
@@ -14,7 +14,35 @@ import { LxdApiResponse } from "types/apiResponse";
 import { LxdOperationResponse } from "types/operation";
 import axios, { AxiosResponse } from "axios";
 import { LxdClusterMember } from "types/cluster";
-import { cephDriver } from "util/storageOptions";
+
+const getClusterAndMemberConfigs = (configs?: LxdStoragePool["config"]) => {
+  if (!configs) {
+    return {};
+  }
+
+  const memberSpecificConfigKeys = new Set([
+    "source",
+    "size",
+    "zfs.pool_name",
+    "lvm.thinpool_name",
```

Comment:
include lvm configs here as well since we will have yaml editing capabilities soon